model/models.urdf.xacro

```diff
@@ -66,6 +66,51 @@
 		<origin xyz="0 0 0.0145" rpy="0 0 0"/>
 	</xacro:property>
 
+<!-- http://www.ati-ia.com/products/ft/ft_models.aspx?id=Gamma -->
+<xacro:property name="gamma">
+			<inertial>
+				<mass value="0.255"/>
+				<origin xyz="0 0 0.01665"  />
+				<xacro:cylinder_inertia_def radius="0.0377" length="0.0333" mass="0.255"/>
+			</inertial>
+			<visual>
+				<!-- origin xyz="0 0 0.00725" rpy="0 0 0"/ -->
+				<origin xyz="0 0 0" rpy="0 0 0"/>
+				<geometry>
+					<!-- cylinder radius="0.0085" length="0.0145"/ -->
+					<mesh filename="package://force_torque_sensor/model/meshes/nano17.stl" scale="4.4 4.4 1"/>
```

Comment:
I tried to find the stl on the official web site, but I failed.

2015-12-21 10:57 GMT+01:00 Carlos J. Rosales <notifications@github.com>:

> In model/models.urdf.xacro
> <https://github.com/CentroEPiaggio/force-torque-sensor/pull/2#discussion_r48130715>
> :
>
> > @@ -66,6 +66,51 @@
> >  		<origin xyz="0 0 0.0145" rpy="0 0 0"/>
> >  	</xacro:property>
> >
> > +<!-- http://www.ati-ia.com/products/ft/ft_models.aspx?id=Gamma -->
> > +<xacro:property name="gamma">
> > +			<inertial>
> > +				<mass value="0.255"/>
> > +				<origin xyz="0 0 0.01665"  />
> > +				<xacro:cylinder_inertia_def radius="0.0377" length="0.0333" mass="0.255"/>
> > +			</inertial>
> > +			<visual>
> > +				<!-- origin xyz="0 0 0.00725" rpy="0 0 0"/ -->
> > +				<origin xyz="0 0 0" rpy="0 0 0"/>
> > +				<geometry>
> > +					<!-- cylinder radius="0.0085" length="0.0145"/ -->
> > +					<mesh filename="package://force_torque_sensor/model/meshes/nano17.stl" scale="4.4 4.4 1"/>
>
> You are using the same visual as the nano17!
> I'll try to find the CAD of the gamma sensor and send it to you to use the
> correct one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CentroEPiaggio/force-torque-sensor/pull/2/files#r48130715>
> .
>